ui/src/views/Provisioners/ViewWorkers/index.jsx

```diff
@@ -59,6 +59,23 @@ export default class ViewWorkers extends Component {
     selectedAction: null,
   };
 
+  componentDidMount() {
+    const { location } = this.props;
+
+    location.search === '?quarantined' ? this.deepLinkFilterDropdown() : null;
```

Comment:
Let's try to use the `qs` package similar to how it's done in the other views.

https://github.com/taskcluster/taskcluster/blob/8319d41e940341f656477e72d912916fc0301b5d/ui/src/views/PulseMessages/index.jsx#L14

Reply:
Okay, alright. So does that mean that I should use bindings to implement deep linking instead of my approach?

Reply:
Bindings is something only relevant to the pulse-message view. In our view, we can do `?filterBy=quarantined`.

Reply:
Yes, I have pushed changes. Can you take a look at it please? 
@helfi92